.eleventy.js

```diff
@@ -298,10 +353,31 @@ ${text.trim()}
 		return commaNumber(num);
 	});
 
+	eleventyConfig.addFilter("friendlyAuthorName", function(name) {
+		if(name && name.startsWith("twitter:")) {
+			return `<em>${name.substring("twitter:".length)}</em>`;
+		}
+		return name;
+	});
+
 	eleventyConfig.addFilter("displayPrice", function(num) {
 		return new Intl.NumberFormat('en-US', { style: 'currency', currency: 'USD', minimumFractionDigits: 0 }).format(num);
 	});
 
+	eleventyConfig.addFilter("displayUrl", function(url, keepWww = false) {
+		if(!keepWww) {
+			url = url.replace("https://www.", "");
```

Comment:
Not sure what this is used for, but does it (and L371 below) need to support `http://` (non httpS) links as well?